demos/python_demos/object_detection_demo_faceboxes/object_detection_demo_faceboxes.py

```diff
@@ -0,0 +1,132 @@
+"""
```

Comment:
Please add shebang line and set Linux execution permissions for that file.

Reply:
Done

Reply:
You should also set Linux execution permissions for that file.

Reply:
Done